docs/origin-story.rst

```diff
@@ -138,3 +138,13 @@ over 1,500 :doc:`meetup </meetups/index>` members that have joined meetup groups
 
 The community keeps growing larger and larger,
 and we're happy to welcome everyone into this wonderful group of documentarians.
+
+For a video testimonial, see this interview of Why You Should Doc Code with Write the Docs' Eric Holscher & Marcia Riefer Johnston:
+
+.. raw:: html
+
+    <div class="talk">
+      <div class="embed-container">
+        <iframe src="https://www.youtube.com/embed/j6rQpO_6XUU" frameborder="0" allowfullscreen></iframe>
```

Comment:
```suggestion
        <iframe src="https://www.youtube-nocookie.com/embed/j6rQpO_6XUU" frameborder="0" allowfullscreen></iframe>
```

ref https://stackoverflow.com/questions/42555450/how-to-embed-youtube-videos-without-cookies